src/flux_workarounds.jl

```diff
@@ -0,0 +1,57 @@
+using Flux: BatchNorm, InstanceNorm, GroupNorm, Params, trainable
+using Base: IdSet
+export weights, loadweights!
+
+"""
+    LegolasFlux.other_weights(layer) -> Vararg{Array}
+
+Given a layer with params that are not captured by `Flux.trainable`, produce
+a tuple of arrays corresponding to these parameters (analogous to `Flux.trainable`).
+"""
+function other_weights end
+
+other_weights(layer) = ()
```

Comment:
Re Flux dependency, the only dependency that is strictly necessary here is Functors.jl itself. `Functors.children(layer)` will return _everything_ that layer considers a parameter, so you shouldn't be missing out on anything :)